MobileFilter/sections/specific_web.txt

```diff
@@ -248,6 +248,8 @@ ameblo.jp##._eP0GEh2C
 9ketsuki.info##.adsbygoogle
 sonicch.com###text-127
 sonicch.com###gn_interstitial_area
+||e.z091u.cn^$domain=shenpinwu.com
+||m.sdxlglm.cn^$domain=shenpinwu.com
```

Comment:
Reply of both urls are only "-1". Do they fix that ad there?

Reply:
Yes, they do that. With that rules i can't reproduce ads.